### ISSUE TITLE
Result looks some strange

### ISSUE THREAD

**Issue by gumanchang** [minqukanq/hierarchical-multi-label-text-classification#5]
root - INFO - All Validation set: Loss 0.00198838 | AUC 0.97652 | AUPRC 0.338539
root - INFO - Predict by threshold: Precision 0.716505, Recall 0.196933, F 0.30895
root - INFO - Predict by topK:
root - INFO - Top1: Precision 0.839844, Recall 0.108801, F 0.192646
root - INFO - Top2: Precision 0.686198, Recall 0.177793, F 0.282414
root - INFO - Top3: Precision 0.591112, Recall 0.229735, F 0.330876
root - INFO - Top4: Precision 0.519005, Recall 0.268948, F 0.354299
root - INFO - Top5: Precision 0.463862, Recall 0.300466, F 0.364699

 I use all dataset (from google drive) to train, But it result shows F1 score is too low. Can any suggestions give me?

**Comment by minqukanq** [comment]
Hi, I had the same result. :)
This implementation uses a low learning rate and warmup.
In my experience, it takes a very long time to learn to get a high F1 score.

**Comment by minqukanq** [comment]
https://github.com/minqukanq/hierarchical-multi-label-text-classification/issues/2

**Comment by gumanchang** [comment]
Thanks. So the args is default, and wait more epoch for better result?

**Comment by minqukanq** [comment]
Even in the original paper, when the Patent dataset was used, the f1 score of the final output was 0.3.

It seems like a good way to check the performance of each layer.

And in the original paper, word2vec was used, but I used the embedding layer for convenience.
Perhaps you can get a higher score if you use word2vec.
